shaders/src/main/glsl/samples/310es/squares_intfunctions.frag

```diff
@@ -0,0 +1,139 @@
+#version 310 es
+
+/*
+ * Copyright 2019 The GraphicsFuzz Project Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+precision highp float;
+
+layout(location = 0) out vec4 _GLF_color;
+
+uniform float time;
+
+uniform vec2 resolution;
+
+float h_r;
+float s_g;
+float b_b;
+
```

Comment:
Again, I think the uses are very simple here. But perhaps it is fine to leave one shader with *only* simple uses and see what happens!